src/libraries/Fuzzing/DotnetFuzzing/DotnetFuzzing.csproj

```diff
@@ -9,9 +9,7 @@
     <AppHostSourcePath>$(ArtifactsDir)\bin\win-x64.Debug\corehost\apphost.exe</AppHostSourcePath>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
-    <NoWarn>$(NoWarn);CS1591;IL3000;SYSLIB1054;CA1512;SYSLIB5005;</NoWarn>
```

Comment:
Was this part intentional?

Reply:
Yes, since we set the `IsTestSupportProject` property in  https://github.com/dotnet/runtime/blob/ba28992b5723ed5bee39829876a39b9b411594c9/src/libraries/Fuzzing/Directory.Build.props#L3 these analyzer diagnostics are not hit on build, probably because we suppress warning on test build